src/tests/const.test.js

```diff
@@ -0,0 +1,86 @@
+import { createHeadlessForm } from '../createHeadlessForm';
```

Comment:
> @sandrina-p are you the admin on this repo? Can we change it so that new commits dismiss old reviews?

> if you or @johnstonbl01 can give this a "reapproval" i think we can merge now, green internally now!


I hate github for not allowing threads in "standalone" comments, so I'm creating this new comment:

Reply:
Yes, I was the only admin, I changed that now, pls double-check. I also added a branch rule protection to `main` to require re-approvals after commits.

Reply:
Looks good, thank you!